### ISSUE TITLE
Modal composer buttons on bottom are not sticky to bottom on 4k screen

### ISSUE THREAD

**Issue by miaulalala** [nextcloud/mail#6273]

**Comment by JuliaKirschenheuter** [comment]
@ChristophWurst , @miaulalala , @GretaD could you reproduce it? If yes - i will make a description for this ticket. If not - i will close this one. I can not reproduce it anymore.

**Comment by GretaD** [comment]
yes, this is fixed for 4k/big/small screen https://github.com/nextcloud/mail/pull/6345

**Comment by GretaD** [comment]
what the heck is that, how did you reproduce it?

**Comment by GretaD** [comment]
i can reproduce it if i make my very big screen 60% small..but its the a sticky button problem, but the modal changing size when its not supposed to.

**Comment by GretaD** [comment]
for context, the attachment also are positioned like this.
![bigscreen](https://user-images.githubusercontent.com/12728974/166943465-def66776-7eec-4aec-b5f3-b40164ed472a.png)
